string name
int32 dof
bool servoState
bool clawState
float32[] terminal
float32[] joints
float32[] homogeneousMatrix

float32[] float_array
int32 integer_value